frontend/sportsmatch-app/src/components/RateGameComponent.tsx

```diff
@@ -0,0 +1,211 @@
+import {
+  ApiError,
+  EventDTO,
+  OpenAPI,
+  RatingControllerService,
+  RatingDTO,
+} from '../generated/api'
+import '../styles/RateGameComponent.css'
+import Avatar from './Avatar'
+import Rating from './Rating'
+import { ChangeEvent, FormEvent, useEffect, useState } from 'react'
+
+interface Props {
+  toggle: () => void
+}
+
+export default function RateGameComponent(p: Props) {
+  const sampleEvent: EventDTO = {
```

Comment:
I think the backend is implemented for this, so we don't need a sample event right?